dev/com.ibm.ws.security.oidc.client_fat.2/fat/src/com/ibm/ws/security/openidconnect/client/fat/IBM/OidcPropagationConsumeUserinfoTests.java

```diff
@@ -0,0 +1,1213 @@
+                    }
+                } else {
+                    expectations = vData.addExpectation(expectations, Constants.INVOKE_RS_PROTECTED_RESOURCE, Constants.RESPONSE_FULL, Constants.STRING_DOES_NOT_MATCH, "Did not see the claim " + parm.getName() + " in the response.", null, "\"" + parm.getName() + "\":\"" + parm.getValue() + "\"");
+                }
+            }
+
+            if (!userinfoWillBeUsed) {
+                if (isJWS) {
+                    expectations = validationTools.addMessageExpectation(genericTestServer, expectations, Constants.INVOKE_RS_PROTECTED_RESOURCE, Constants.TRACE_LOG, Constants.STRING_CONTAINS, "Client trace.log should contain a message indicating that there is a signature mismatch", MessageConstants.CWWKS1777E_SIG_ALG_MISMATCH);
+                } else {
+                    expectations = validationTools.addMessageExpectation(genericTestServer, expectations, Constants.INVOKE_RS_PROTECTED_RESOURCE, Constants.TRACE_LOG, Constants.STRING_CONTAINS, "Client messages.log should contain a message indicating that there is a signature mismatch", MessageConstants.CWWKS6056E_ERROR_EXTRACTING_JWS_PAYLOAD_FROM_JWE);
+                }
+            }
+
+        }
+
+        return expectations;
+    }
+
+    /**
+     * Create a jwt token with extra parms - this will be used to generate the the token saved and used by the test "userinfo"
+     * endpoint.
+     * The extra parm "defaultExtraClaim" will be used to show that the runtime is picking up info from userinfo.
+     *
+     * @param parms
+     *            - current list of parms to add values to
+     * @param builderId
+     *            - the builderId that will be used (passed to method for use in determining if extra EC parms are needed)
+     * @return - updated list of parms to add to token
+     * @throws Exception
+     */
+    protected List<NameValuePair> createExtraParms(List<NameValuePair> parms, String builderId) throws Exception {
+        if (parms == null) {
+            parms = new ArrayList<NameValuePair>();
+        }
+        parms.add(new NameValuePair("sub", "testuser"));
+        parms.add(new NameValuePair("defaultExtraClaim", "someValue"));
+        parms = setParmsForECWorkaround(parms, builderId);
+        return parms;
+
+    }
+
+    /**
+     * Creates a token using the passed builder along with any parms passed
+     *
+     * @param builderId
+     *            - the builder id to use to build the token
+     * @param parms
+     *            - extra parms if any
+     * @return - a token built using the builder id and extra parms passed
+     * @throws Exception
+     */
+    protected String createTokenWithSubjectPlus(String builderId, List<NameValuePair> parms) throws Exception {
+
+        String jwtToken = actions.getJwtTokenUsingBuilder(_testName, testOPServer.getServer(), builderId, parms);
+        Log.info(thisClass, _testName, jwtToken);
+
+        return jwtToken;
+    }
```

Comment:
Is this method really needed since all it does is call `actions.getJwtTokenUsingBuilder()`?

Reply:
put the call to getJwtTokenUsingBuilder in line and will deliver with changes to share code between oidc and social